### DISCUSSION TITLE
Applicazione jw per microsoft

### DISCUSSION THREAD

**Discussion by simomaca** [AntonyCorbett/OnlyM#440] [General]
Ciao a tutti , mi sembrate molto esperti perciò se mi permettete vi chiedo un consiglio .
Ho applicazione jw come secondo monitor non si apre sempre sul monitor del podio ma su quello del contatempo . 
Ho più volte ricreato i registro di sistema di windows scollegato i monitor e ricollegando come primo quello sul podio e come secondo il timer ... riavviando il pc mantiene l'impostazione magari diversi giorni ma poi senza apparenti motivi riinverte i monitor. avete idee ? 

[English] Hello everyone, you seem very experienced so if you allow me I ask you for advice. I have JW Library in dual monitor mode, but it does not always open on the podium monitor but on the timer monitor. I have repeatedly recreated the windows registry, disconnected the monitors and reconnected the one on the podium as the first and the timer as the second. Restarting the PC it keeps the setting for several days but then for no apparent reason re-inverts the monitors. do you have any ideas?

**Comment by Wistula** [comment]
JW app throws screen to right. I am not an expert, but maybe the screen loses the signal and changes the order in Windows. If this happens, check the order and position of the screens in the manager

**Comment by simomaca** [comment]
Grazie mille . Hai ragione . Lo mettevo sopra e lo perdevo . A destra no .
Grazie tanto .



Saluti
Simone Macaluso
________________________________
